common/class.tx_dlf_document.php

```diff
@@ -1992,6 +2000,8 @@ protected function _getFileGrps() {
             // Are there any fulltext files available?
             if (!empty($extConf['fileGrpFulltext']) && in_array($extConf['fileGrpFulltext'], $this->fileGrps)) {
 
+                $fft = $this->mets->xpath("./mets:fileSec/mets:fileGrp[@USE='FULLTEXT']/mets:file[1]");
```

Comment:
The `@USE` attribute of the fulltext file group is configurable, so it must not be hard-coded here.

```suggestion
                $fft = $this->mets->xpath('./mets:fileSec/mets:fileGrp[@USE="' . $extConf['fileGrpFulltext'] . '"]/mets:file[1]');
```